### PR TITLE
resolved issue with law2vec document  fetching

### PR DESCRIPTION
**Pull request by DanCh11**

### CODE REVIEW

File: sem_covid/services/pwdb_knn_experiments.py [hunk 1]
```diff
@@ -22,75 +21,45 @@
 from sem_covid.services.pwdb_base_experiment import PWDBBaseExperiment
 from sem_covid.services.sc_wrangling.mean_vectorizer import MeanEmbeddingVectorizer
 from sem_covid.services.store_registry import StoreRegistry
+from sem_covid.services.data_registry import LanguageModel
```
Comment (costezki) [inline sem_covid/services/pwdb_knn_experiments.py:24]: I am no longer sure that the old ml experiments and what you have developed with KNN and SVM classifiers are necessary any longer. Can you check this with Stefan, please?

File: sem_covid/services/pwdb_svm_experiment.py [hunk 2]
```diff
@@ -46,8 +45,8 @@ def __init__(self, minio_model_adapter, minio_adapter, requests, **kwargs):
     def model_training(self, *args, **kwargs):
         train_test_dataset = pickle.loads(self.minio_adapter.get_object(PWDB_TRAIN_TEST))
 
-        law2vec = LanguageModel.LAW2VEC.fetch()
-        law2vec_format = KeyedVectors.load_word2vec_format(law2vec, binary=True)
+        law2vec_path = LanguageModel.LAW2VEC.path_to_local_cache()
```
Comment (costezki) [inline sem_covid/services/pwdb_svm_experiment.py:48]: please talk to Stefan, if these experiments are needed any longer.

**Review by costezki — APPROVED** [review]
before merge, check the status of these experiments and cleanup if they are no longer needed.

File: sem_covid/services/pwdb_knn_experiments.py (hunk 1 above)
Comment (DanCh11) [reply sem_covid/services/pwdb_knn_experiments.py:24]: I deleted them in `feature/sc-335` and it will be no longer in our project after the merge

**Pull request closed without merging**